WordPress/src/main/java/org/xmlrpc/android/XMLRPCUtils.java

```diff
@@ -289,6 +292,14 @@ private static String verifyXmlrpcUrl(final String siteUrl, final String httpUse
                     // Endpoint found and works fine.
                     return url;
                 }
+            } catch (XMLRPCUtilsException e) {
+                if (e.kind == XMLRPCUtilsException.Kind.ERRONEOUS_SSL_CERTIFICATE ||
+                    e.kind == XMLRPCUtilsException.Kind.HTTP_AUTH_REQUIRED ||
+                    e.kind == XMLRPCUtilsException.Kind.MISSING_XMLRPC_METHOD) {
```

Comment:
hey @daniloercoli , thanx for pushing these extra commits!

Can you share some insight on why you opt to let `MISSING_XMLRPC_METHOD` bubble up? I mean, I understand the other two as those cause a prompt for the user, but what about MISSING_XMLRPC_METHOD? Why not let the loop continue with the rest of the URLs even though it will probably lead to the same error for all URLs?